Sources/BraintreeApplePay/BTConfiguration+ApplePay.swift

```diff
@@ -57,6 +62,6 @@ extension BTConfiguration {
             }
         }
 
-        return supportedNetworks
+        return supportedNetworks.isEmpty ? [] : supportedNetworks
```

Comment:
It looks like we set `supportedNetworks` to `[]` earlier. Do we need this logic or should that catch this case?